### DISCUSSION TITLE
Google Scholar citation number doesn't update?

### DISCUSSION THREAD

**Discussion by zjysteven** [alshedivat/al-folio#2575] [Q&A] [answered]
Hello, thank you for this great template. I have set things up to track my publications on the homepage and I'm able to get google scholar badges next to each publication with the citation number showing in the badge. However, the citation number hasn't changed since I set them up (while I do get new citations over the past couple months). I was thus wondering if there might be something wrong on my end, or it is expected. Thank you!

**Comment by george-gca** [comment, marked as answer]
This is expected. Google scholar badge is not an official Google product, rather is a solution that I implemented using a scrapper that fetches this information at build time.

What this means is, unless you make some changes that trigger your site to be built and deployed again, the value on the Google Scholar badge will not change.
